search/src/main/java/edu/artic/search/SearchBaseViewModel.kt

```diff
@@ -174,4 +176,36 @@ open class SearchBaseViewModel @Inject constructor(
     open fun onClickSeeAll(header: Header) {
 
     }
+
+    /**
+     * Returns suggested artworks.
```

Comment:
Perhaps `Returns suggested artworks, in order.`? I'm kind ambivalent about which we go for here.